feedbacks/text_analyzer.py

```diff
@@ -1,15 +1,25 @@
+from threading import activeCount
 import joblib
 import nltk
+import ssl
 from nltk import TokenSearcher, word_tokenize
 from unidecode import unidecode
 
+try:
+    _create_unverified_https_context = ssl._create_unverified_context
+except AttributeError:
+    pass
+else:
+    ssl._create_default_https_context = _create_unverified_https_context
+
 nltk.download("punkt")
 
 TAGGER = joblib.load("POS_tagger_brill.pkl")
 
 
 class TextAnalyzer:
     GENDER_MARKS = {"os", "ores", "ões", "ns", "ãos"}
+    DOUBLE_CHECK_WORDS = {"pessoa", "pessoas"}
```

Comment:
@felipmartins o nome `DOUBLE_CHECK_WORDS` me parece pouco explícito... tem outra sugestão?

Pensei em tirar o `"pessoa"` também, porque atualmente a API só acusa problemas com plurais. O que acha?

Reply:
```suggestion
    NEUTRAL_MARK = {"pessoas"}
```

O que acha? @vbuxbaum  me veio essa outra nomenclatura na cabeça agora, fica à vontade para sugerir algo que considere melhor 😄

Reply:
Legaaaal ❤️